src/app/core/basic-datatypes/date/date-range-filter/date-range-filter.component.ts

```diff
@@ -20,6 +27,12 @@ export class DateRangeFilterComponent<T extends Entity> implements OnChanges {
 
   @Input() filterConfig: DateFilter<T>;
 
+  /**
+   * Emits an array with two elements, the selected from and to dates as string
+   */
+  @Output() selectedOptionChange = new EventEmitter<string[]>();
```

Comment:
- this doesn't seem to be used/emitted now (and does almost the same thing as dateRangeChange)?